composer/utils/dist.py

```diff
@@ -519,7 +519,10 @@ def initialize_dist(device: Union[str, Device], timeout: float = 300.0):
 
     dist_env_vars_match_defaults = all(os.environ.get(k, v) == v for (k, v) in dist_env_var_defaults.items())
 
-    if dist_env_vars_match_defaults:
+    if device_obj.dist_backend == 'xla':
+        # XLA initialization requires the init_method to be set
+        dist.init_process_group(device_obj.dist_backend, init_method='xla://')
```

Comment:
Should we add a version.parse here for the different pytorch options?

Reply:
Does the latest change look OK? Looks like this is really dependent on the PyTorch XLA version being 2.1 or greater. I believe the versions are parallel to pytorch, but its better to check the correct module.

Reply:
looks good!